src/dcqc/mixins.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC, abstractmethod
+from typing import Union
+
+SerializedObject = dict[str, Union[str, int]]
+
+
```

Comment:
I say that this is the appropriate way to provide this feature in a lot of classes.